packages/platform/atoms/booker/BookerWebWrapper.tsx

```diff
@@ -219,6 +225,9 @@ export const BookerWebWrapper = (props: BookerWebWrapperAtomProps) => {
       schedule={schedule}
       verifyCode={verifyCode}
       isPlatform={false}
+      areInstantMeetingParametersSet={areInstantMeetingParametersSet}
+      userLocale={session?.user.locale}
+      i18nLocales={i18nLocales}
```

Comment:
Hmm passing a prop doesn't seem optimal. I'd suggest importing it from `i18n.json` where it's needed